x-pack/plugins/security_solution/public/management/pages/policy/types.ts

```diff
@@ -98,7 +98,9 @@ export interface PolicyArtifactsState {
   /** A list of trusted apps going to be updated  */
   trustedAppsToUpdate: AsyncResourceState<PostTrustedAppCreateResponse[]>;
   /** Represents if there is any trusted app existing  */
-  doesAnyTrustedAppExists: AsyncResourceState<boolean>;
+  doesAnyTrustedAppExists: AsyncResourceState<GetTrustedAppsListResponse>;
+  /** Represents if there is any trusted app existing assigned to the policy (wuthout filters)  */
+  hasTrustedApps: AsyncResourceState<GetTrustedAppsListResponse>;
```

Comment:
This probably should be named `hasTrustedAppsForPolicy`.

Reply:
I thought that, but as this is inside the policy context an other props doesn't contain the word `policy` I went for that name. Does that make sense to you?

Reply:
It does. It's just that `hasTrustedApps` and `doesAnyTrustedAppExists` kind of seem alike.